packages/dart_frog_test/lib/src/dart_frog_test.dart

```diff
@@ -0,0 +1,53 @@
+import 'dart:async';
+
+import 'package:dart_frog/dart_frog.dart';
+import 'package:meta/meta.dart';
+import 'package:mocktail/mocktail.dart';
+import 'package:test/test.dart';
+
+class _MockRequestContext extends Mock implements RequestContext {}
+
+class _MockRequest extends Mock implements Request {}
+
+/// A test that can be used to test a Dart Frog handler.
+@isTest
+void testDartFrog(
+  String message, {
+  required String url,
+  required FutureOr<Response> Function(RequestContext request) onRequest,
+  HttpMethod method = HttpMethod.get,
+  Map<String, String>? headers,
+  dynamic requestJson,
+  String? requestBody,
```

Comment:
I'm not sure about including parameters for the Request. Maybe, would it be better if we allow the user to provide a `request` and they can do whatever they want with it?

I'm not sure, I might need to give it a shower thought 🚿

Reply:
My reasoning behind having the fields of the Request received by the method was based on my incorrect assumption that instantiating a `Request` class was hard, which by now I see that it is not the case, so yes, this suggestion makes total sense! I am gonna refactor it!